content/en/docs/concepts/services-networking/service.md

```diff
@@ -91,32 +91,32 @@ spec:
     targetPort: 9376
 ```
 
-This specification will create a new Service object named “my-service” which
-targets TCP port 9376 on any Pod with the `"app=MyApp"` label.
+This specification creates a new Service object named “my-service”, which
+targets TCP port 9376 on any Pod with the `app=MyApp` label.
 
-This Service will also be assigned an IP address (sometimes called the "cluster IP"),
-which is used by the service proxies
+This Service also assigns an IP address (sometimes called the "cluster IP"),
+which is used by the Service proxies
 (see [Virtual IPs and service proxies](#virtual-ips-and-service-proxies) below).
 
-The controller for the Service selector will continuously scan for Pods that
-match its selector, and will then POST any updates to an Endpoint object
+The controller for the Service selector continuously scans for Pods that
+match its selector, and then POSTs any updates to an Endpoint object
 also named “my-service”.
 
 {{< note >}}
-A Service can map _any_ incoming `port` to a `targetPort`.  By default, and
-for convenience, the `targetPort` will be set to the same value as the `port`
+A Service can map _any_ incoming `port` to a `targetPort`. By default and
+for convenience, the `targetPort` is set to the same value as the `port`
 field.
 {{< /note >}}
 
 Port definitions in Pods have names, and you can reference these names in the
-targetPort attribute of a Service. This will work even if there are a mixture
-of Pods in the Service, with the same network protocol available via different
-port numbers but a single configured name.
+`targetPort` attribute of a Service. This works even if there is a mixture
+of Pods in the Service using a single configured name, with the same network
+protocol available via different port numbers.
 This offers a lot of flexibility for deploying and evolving your Services.
-For example, you can change the port number that pods expose in the next
+For example, you can change the port number the Pods are exposed in the next
```

Comment:
This needs to be "that Pods expose". And we should probably say "numbers".

you can change the port numbers that Pods expose in the next version